general-concepts/tree/text.xml

```diff
@@ -38,6 +38,24 @@ The basic layout of the portage tree is as follows:
   <li>Eclasses directory, <c>eclass/</c></li>
   <li>Licenses directory, <c>licenses/</c></li>
   <li>
+  Metadata directory, <c>metadata/</c>. Most of the listed contents are not
+  included in the main Git tree, but are generated by the
+  <uri link="::general-concepts/git-to-rsync">Git to Rsync</uri> process or
+  included there from other repositories.
+  <ul>
+    <li>
+    Various control files, for example <c>layout.conf</c>, <c>timestamp</c>
+    and <c>projects.xml</c>. Only <c>layout.conf</c> exists in the main Git tree.
+    </li>
+    <li>
+    XML validation files in <c>metadata/dtd/</c> and <c>metadata/xml-schema/</c>.
+    </li>
+    <li>
+    News updates in <c>metadata/glsa/</c> and <c>metadata/news/</c>.
```

Comment:
`Security advisories in <c>metadata/glsa/</c> and news items in <c>metadata/news/</c>` How does this sound?